data/functions.sql

```diff
@@ -933,3 +933,10 @@ BEGIN
   RETURN mz_building_height(tags->'height', tags->'building:levels') * way_area;
 END
 $$ LANGUAGE plpgsql IMMUTABLE;
+
+CREATE OR REPLACE FUNCTION trim_nz_sh(label TEXT)
```

Comment:
Add comment here explaining that this trims the "SH" from New Zealand State Highway road shield refs.